http-gateway/service/httpApi.go

```diff
@@ -120,6 +117,11 @@ func NewHTTP(requestHandler *RequestHandler, authInterceptor kitHttp.Interceptor
 	r.HandleFunc(uri.WsStartDevicesObservation, requestHandler.startDevicesObservation).Methods(http.MethodGet)
 	r.HandleFunc(uri.WsStartDeviceResourcesObservation, requestHandler.startDeviceResourcesObservation).Methods(http.MethodGet)
 
+	// serve www directory
+	if requestHandler.config.WWWDirectory != "" {
```

Comment:
UI will be always embedded in the http gateway container. It's more clean to use boolean value to set if the UI should be hosted or not.